src/components/PanelV2/__tests__/PanelV2.spec.js

```diff
@@ -3,38 +3,121 @@
  * @copyright IBM Security 2019
  */
 
-import { mount } from 'enzyme';
-import React, { Fragment } from 'react';
-
-import { label, onClick } from '../_mocks_';
+import { cleanup, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
 
 import { Button, PanelV2, PanelContent } from '../../..';
 
 describe('PanelV2', () => {
-  it('renders', () => {
-    const wrapper = mount(
+  afterEach(cleanup);
+
+  test('should render as expected', async () => {
+    render(
       <PanelV2
-        title={label}
-        subtitle={label}
-        closeButton={{ onClick }}
-        renderFooter={() => (
-          <Fragment>
-            <Button
-              id="example-secondary-button"
-              kind="secondary"
-              onClick={onClick}
-            >
-              Close
-            </Button>
-            <Button id="example-primary-button" onClick={onClick}>
-              Add
-            </Button>
-          </Fragment>
-        )}
+        title="test title"
+        subtitle="test subtitle"
+        closeButton={{
+          label: 'test close',
+        }}
+        renderFooter={() => <Button>test footer button</Button>}
       >
-        <PanelContent>{label}</PanelContent>
+        <PanelContent>test content</PanelContent>
       </PanelV2>
     );
-    expect(wrapper).toMatchSnapshot();
+    await expect(document.body).toMatchSnapshot();
```

Comment:
For this test, what exactly are you wanting to test for that can't be achieved with specific assertions or just looking at a storybook story? The reason I'm asking is that snapshots are so challenging to maintain and identify when real bugs appear. We've started to move away from them.

Reply:
@diego-codes I was keeping this one out of habit, to be honest. 😅 But you make a good point so I'll go ahead and remove it.

Reply:
@diego-codes I'd like to understand more about your decisions to move away from snapshot tests if you wouldn't mind elaborating?

I personally still find value from them as long as they're being used in moderation and in conjunction with a more comprehensive testing suite, but I'd love to hear more about the challenges you've faced with them!